src/middlewared/middlewared/plugins/chart_releases_linux/scale_workload.py

```diff
@@ -67,6 +68,13 @@ async def scale(self, job, release_name, options):
                     f'{release_name!r} cannot be started as it is consuming following host path(s) '
                     f'which are locked: {", ".join(release["resources"]["locked_host_paths"])}'
                 )
+            elif non_existent_host_paths := [path for path in release['resources']['host_path_volumes'] if not (
```

Comment:
It's not really a big deal right now since most apps are only going to have 1 or 2 host paths configured for an app. However, spinning up background threads for every path seems a little much. Would be better, imo, to have a synchronous function that does `os.path.exists` calls and only 1 thread get used. Doesn't need to be fixed in this PR but something to consider in future.